backend/prisma/schema.prisma

```diff
@@ -20,4 +20,17 @@ model User {
     name              String?
     createdAt         DateTime  @default(now())
     updatedAt         DateTime  @default(now())
+
+    journals          Journal[]
+}
+
+model Journal {
+    id         String       @id @default(cuid())
+    userId     String
+    title      String
+    content    String?
+    createdAt  DateTime  @default(now())
+    updatedAt  DateTime  @default(now())
```

Comment:
Updated at should be `@updatedAt` instead of `@default(now())`. This will maek sure it gets automatically updated each time you are updating your db

Change all updatedAt in your prisma schema

Reply:
Done!